llumnix/backends/profiling.py

```diff
@@ -205,29 +202,25 @@ def get_latency_mem(backend_type: BackendType, profiling_database: ProfilingData
     import argparse
     parser = argparse.ArgumentParser()
     parser.add_argument("--database", type=str, default="profiling.pkl")
-    parser.add_argument("--log-path", type=str)
-    parser.add_argument("--model", type=str)
+    parser.add_argument("--log-csv-path", type=str, required=True)
+    parser.add_argument("--model", type=str, help="filename of your model, like 'Meta-Llama-3-8B-Instruct'")
     parser.add_argument("--gpu", type=str, default="a10")
     parser.add_argument("--tp", type=int, default=1)
     parser.add_argument("--pp", type=int, default=1)
     parser.add_argument("--gpu-memory-utilization", type=float, default=0.9)
     parser.add_argument("--block-size", type=int, default=16)
     parser.add_argument("--max-num-batched-tokens", type=int, default=8000)
-    parser.add_argument("--num-gpu-blocks", type=int, default=0)
+    parser.add_argument("--num-gpu-blocks", type=int, required=True, help="kv cache blocks number")
```

Comment:
if --num-gpu-blocks is required, why --gpu is not required?

Reply:
Maybe remove --gpu is better? It's not a engine related args.